src/app/state/orderBookSlice.test.ts

```diff
@@ -40,19 +40,19 @@ const MOCK_BUYS = [
 ];
 
 describe("toOrderBookRowProps", () => {
-  it(`returns ${MAX_ROWS} rows if the input is empty`, () => {
+  it(`returns 11 rows if the input is empty`, () => {
```

Comment:
Orderbook rows amount will change in the recent redesign. I would prefer not hardcoding the number. 
Suggestion: "returns correct amount of rows if input is empty".